packages/@aws-cdk/aws-synthetics/lib/runtime.ts

```diff
@@ -177,10 +187,52 @@ export class Runtime {
    * - Selenium version 3.141.0
    * - Chromium version 83.0.4103.0
    *
-   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_python_selenium.html
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_python_selenium.html#CloudWatch_Synthetics_runtimeversion-syn-python-selenium-1.0
    */
   public static readonly SYNTHETICS_PYTHON_SELENIUM_1_0 = new Runtime('syn-python-selenium-1.0', RuntimeFamily.PYTHON);
 
+  /**
+   * `syn-python-selenium-1.1` includes the following:
+   * - Lambda runtime Python 3.8
+   * - Selenium version 3.141.0
+   * - Chromium version 83.0.4103.0
+   *
+   * New Features:
+   * - **Custom handler function**: You can now use a custom handler function for your canary scripts.
+   * - **Configuration options for adding metrics and step failure configurations**: These options were already available in runtimes for Node.js canaries.
+   * - **Custom arguments in Chrome**: You can now open a browser in incognito mode or pass in proxy server configuration.
+   * - **Cross-Region artifact buckets**: A canary can store its artifacts in an Amazon S3 bucket in a different Region.
```

Comment:
I don't think we need this here

Reply:
I can get these removed but, I saw the new features being mentioned within the comments for few runtime versions so added them to make it consistent.

Reply:
ah, ok. Leave them then, we should be consistent.